guides/17_Migration_to_3_0.md

```diff
@@ -159,13 +159,86 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+### `MeetingSessionPOSTLogger` to `POSTLogger`
+
+With Amazon Chime SDK for JavaScript v3, we have removed `MeetingSessionConfiguration` dependency from the `MeetingSessionPOSTLogger`. The `MeetingSessionPOSTLogger` is now renamed to `POSTLogger`.
+
+Builders can add optional fields to `POSTLogger` which will be included when `POSTLogger` sends the HTTP POST logs to builder provided URL:
+
+1. HTTP POST request `headers`.
+2. Informational `metadata`. For example, `meetingId`, `attendeeId`, `appName`. This `metadata` depends on builders and `POSTLogger` will include this `metadata` as part of the request body when making the HTTP POST requests to builder provided URL.
+
+#### Constructor
+
+```js
+
+// You need responses from server-side Chime API. See below for details.
+const meetingResponse = // The response from the CreateMeeting API action.
+const attendeeResponse = // The response from the CreateAttendee API action.
+
+// Before in 2.x
+const meetingSessionConfiguration = new MeetingSessionConfiguration(meetingResponse, attendeeResponse);
```

Comment:
nit: Syntax highlighting is broken... Too 

<img width="552" alt="image" src="https://user-images.githubusercontent.com/36240708/160725388-ded7a134-e80a-4532-b9cc-f0b222ad43ec.png">